print-dhcp6.c

```diff
@@ -784,6 +790,43 @@ dhcp6opt_print(netdissect_options *ndo,
 			ND_PRINT(")");
 			break;
 
+		case DH6OPT_BOOTFILE_URL:
+			if (optlen < 2) {
+				ND_PRINT(" ?)");
+				break;
+			}
+			tp = (const u_char *)(dh6o + 1);
+			ND_PRINT(" ");
+			nd_printjnp(ndo, tp, optlen);
+			ND_PRINT(")");
+			break;
+
+		case DH6OPT_SZTP_REDIRECT:
+		case DH6OPT_USER_CLASS:
+			ND_PRINT(" ");
+			tp = (const u_char *)(dh6o + 1);
+			first_list_value = TRUE;
+			remainder_len = optlen;
+			while (tp < cp + sizeof(*dh6o) + optlen - 2) {
+				if (first_list_value == FALSE) {
+					ND_PRINT(",");
+				}
+				first_list_value = FALSE;
+				subopt_len = GET_BE_U_2(tp);
+				if (subopt_len > remainder_len-2) {
+					break;
+				}
+				tp += 2;
+				nd_printjnp(ndo, tp, subopt_len);
+				tp += subopt_len;
+				remainder_len -= (subopt_len+2);
+			}
+			if (remainder_len != 0 ) {
+				ND_PRINT(" ?");
```

Comment:
This could be a better "invalid" marker too.

Reply:
Since the next line prints ")", this ends up being "?)" which we've established is (in this file) a commonly-used truncation identifier.  Is it ok to keep this as-is?

Reply:
Yes, as discussed.